site/en/docs/extensions/reference/devtools_recorder/index.md

```diff
@@ -0,0 +1,55 @@
+---
+api: devtools.recorder
+title: chrome.devtools.recorder
+---
+
+See [DevTools APIs summary][1] for general introduction to using Developer Tools APIs.
+
+## Overview
+
+`devtools.recorder` API is a preview feature that allows you to extend the [Recorder panel](/docs/devtools/recorder/) in Chrome DevTools.
+Currently, you can extend only the export feature.
+
+To register an extension plugin, use the `registerRecorderExtensionPlugin` function.
+This function requires a plugin instance that implements the following:
+
+- `stringify` and `stringifyStep` functions
+- `name` and `mediaType` parameters
```

Comment:
That is not technically correct now as the plugin instance does not implement those. Can we rephrase it a bit more?

Reply:
I beg your pardon. Apparently, I didn't get that.